helm/flink-operator/values.yaml

```diff
@@ -41,6 +41,28 @@ jobServiceAccount:
     "helm.sh/resource-policy": keep
   name: "flink"
 
+operatorVolumeMounts:
+  create: false
+  data:
+    - name: flink-userlib
```

Comment:
I would also suggest the `flink-artifacts`.

Reply:
Fixed